run.jl

```diff
@@ -0,0 +1,12 @@
+using ExercismTestReports
+
+const SOLUTION_DIR = ARGS[2]
+const OUTPUT_DIR = ARGS[3]
+
+@debug "SOLUTION_DIR = " SOLUTION_DIR
+@debug "OUTPUT_DIR = " OUTPUT_DIR
```

Comment:
This doesn't do anything in the final image, but may be useful when debugging issues with the image itself.